source/MRViewer/MRSurfaceManipulationWidget.h

```diff
@@ -49,6 +49,8 @@ class MRVIEWER_CLASS SurfaceManipulationWidget :
     MRVIEWER_API void init( const std::shared_ptr<ObjectMesh>& objectMesh );
     /// reset widget state
     MRVIEWER_API void reset();
+    /// set unchangeable region
```

Comment:
please add more details in the comment, and mb notice that patch could still affect region boundary

Reply:
fixed